tests/x11regressions/evince/evince_find.pm

```diff
@@ -0,0 +1,37 @@
+use base "x11test";
+use strict;
+use testapi;
+
+# Case 1436022 - Evince: Find
+sub run() {
+    my $self = shift;
+    x11_start_program("evince " . autoinst_url . "/data/x11regressions/test.pdf");
+
+    send_key "ctrl-f";    # show search toolbar
+    assert_screen 'evince-search-toolbar', 5;
+
+    my $keyword = 'To search for';
+    type_string $keyword;
```

Comment:
Why are you using $keyword variable when it's only used once? Why not
```perl
type_string 'To search for';
```

Am I missing something?